src/wallet/rpc/util.cpp

```diff
@@ -122,16 +122,8 @@ std::string LabelFromValue(const UniValue& value)
     return label;
 }
 
-std::tuple<std::shared_ptr<CWallet>, std::vector<bilingual_str>> LoadWalletHelper(WalletContext& context, UniValue load_on_start_param, const std::string wallet_name)
+void HandleWalletError(const std::shared_ptr<CWallet> wallet, DatabaseStatus& status, bilingual_str& error)
```

Comment:
In 33763d02ef1011ae71143c5e6e313c72d9b790a6 "refactor: Move restorewallet() logic to the wallet section"

It would be easier to review this diff if this refactor were in its own commit.

Reply:
Done in a9f9a83.